std_msgs/Header header
FingerInfo[] fingers
